src/wallet/node_rpc_proxy.cpp

```diff
@@ -138,6 +142,15 @@ boost::optional<std::string> NodeRPCProxy::get_target_height(uint64_t &height) c
   return boost::optional<std::string>();
 }
 
+boost::optional<std::string> NodeRPCProxy::get_immutable_height(uint64_t &height) const
+{
+  auto res = get_info();
+  if (res)
+    return res;
+  height = m_immutable_height;
+  return boost::optional<std::string>();
```

Comment:
The failure case is more commonly `return boost::none;`, but I see nothing else here is doing that so I guess leave it. :disappointed: